CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
 
 ## Neste versjon
 
+- ⚡ **Admin**. Optimaliserte "er du sikker"-popups i adminpanelet.
```

Comment:
Glemte kanskje å nevne hvilke ganger det ikke er relevant å legge til i changelog. Changeloggen er primært for at folk utenfor Index kan se hva som blir gjort. Det er ikke relevant for dem å se mindre refactors som dette, ettersom det ikke er noen forandring for dem - kun bedre kode for oss :)

Reply:
nice👍